flink-runtime-web/web-dashboard/web/css/index.css

```diff
@@ -228,7 +228,6 @@
   overflow: hidden;
   padding: 2px 4px;
   font-size: 14px;
-  -webkit-border-radius: 2px;
```

Comment:
Good catch. I'll take a look at what `gulp` version am I using and check how I can fix it.
Could you please check what version of `gulp` are you using? I don't think it is documented anywhere.